### ISSUE TITLE
Does the plugin work in live preview?

### ISSUE THREAD

**Issue by YousufSSyed** [MamoruDS/obsidian-open-link-with#26]
If not, I'd like it to.

**Comment by MamoruDS** [comment]
Yes, this plugin supports live preview. Clicking on a hyperlink behaves the same way as read mode; clicking on an editing link requires a modifier key such as Ctrl (or Cmd for macOS) to work.

**Comment by YousufSSyed** [comment]
I'm having trouble getting it working, I'm on macOS 13.1. This is one binding I have. I made this link: [link](https://github.com/MamoruDS/obsidian-open-link-with/issues/26), and made this keybinding with Shift which you can see in the attached data.json.
 and tried the following combinations:
* Shift Click
* Shift CMD Click
* Shift CTRL Click
* Shift CTRL CMD Click

and none of them work, they either open the link in the browser, focuses the editor there to edit, or triggers the right click menu.

[data.json.zip](https://github.com/MamoruDS/obsidian-open-link-with/files/11234699/data.json.zip)

**Comment by MamoruDS** [comment]
Sorry for the late reply. I have confirmed that there are issues with custom modifiers in the live preview, which were caused by the refactoring of default click modifier behaviors in the previous version. Thanks for letting me know!

**Comment by YousufSSyed** [comment]
Don't worry about taking your time! I'm sure you only work on this for your free time for free. I appreciate your work!
